src/object/srv_ec_aggregate.c

```diff
@@ -2517,8 +2517,14 @@ cont_ec_aggregate_cb(struct ds_cont_child *cont, daos_epoch_range_t *epr,
 		ec_agg_param->ap_agg_entry.ae_obj_hdl = DAOS_HDL_INVAL;
 	}
 
-	if (ec_agg_param->ap_obj_skipped)
-		D_ERROR("with skipped obj during aggregation.\n");
+	if (ec_agg_param->ap_obj_skipped) {
+		D_DEBUG(DB_EPC, "with skipped obj during aggregation.\n");
+		/* There is rebuild going no, and we can proceed EC aggregate boundry,
+		 * Let's wait for 5 seconds for another EC aggregation.
+		 */
```

Comment:
Current code will sleep at least 2 seconds between two aggregation runs, this patch enlarged it to 5 seconds when hitting 'ap_obj_skipped'?

Reply:
if hae does not proceed, then it will not wait for 2 seconds.          epoch_min = param->ap_start_eph_get(cont);  will not change in this case.

Reply:
Oh, it's a 200ms sleep, see cont_aggregate_interval(), so this patch enlarges 200ms sleep to a 5 seconds sleep?